tests/test_module_doc.py

```diff
@@ -35,12 +35,12 @@
 # Ignored items will not have their default values checked or be included for
 # alphabetical order purposes
 IGNORE_ITEM = [
+    ('arch_updates', 'format'),  # line too long for docstring parsing
     ('screenshot', 'save_path'),  # home dir issue
     ('rate_counter', 'config_file'),  # home dir issue
     ('group', 'format'),  # dynamic depending on click_mode
     ('github', 'format'),  # dynamic
     ('kdeconnector', '_dev'),  # move to __init__ etc
-    ('arch_updates', 'format'),  # dynamic
```

Comment:
I'm not sure this should be here as cleaning up `tests/test_module_doc.py` is not what the PR is about